bundles/paikkatietoikkuna/admin/admin-layeranalytics/LayerAnalyticsDetails.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Table, Space, Spin } from 'antd';
+import { Message } from 'oskari-ui';
+import { ArrowLeftOutlined } from '@ant-design/icons';
+
+import 'antd/es/table/style/index.js';
+
+const dateLocale = 'fi-FI';
+const localeDateOptions = {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit'
+};
+
+
+// timestamp formatting copied from admin-layereditor in oskari-frontend
+// TO-DO: Move both to helper class
+const formatTimestamp = (timestamp) => {
+    let date;
+    if (typeof timestamp !== 'undefined') {
+        date = new Date(timestamp);
+    }
+    return formatTime(date) + ' ' + formatDate(date);
+};
+const formatDate = (date) => {
+    if (typeof date === 'undefined') {
+        return '--.--.----';
+    }
+    return date.toLocaleDateString(dateLocale, localeDateOptions);
+};
+
+const formatTime = (date) => {
+    if (typeof date === 'undefined') {
+        return '--:--:--';
+    }
+    return date.toLocaleTimeString(dateLocale).replace(/\./g, ':');
+};
+
+const generateToScaleURL = (stack) => {
```

Comment:
generateLink() would be a better name for this. Remember to add stack.z as zoomLevel param.

Reply:
Changed.